baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -176,6 +252,14 @@ def get_snapshot_mode_query(modes):
     validator=Validator([required_field_validator, reachable_url_validator()]),
 )
 
+snapshot_sha256_query = Step(
+    id="snapshot_sha256",
+    prompt="Provide the sha256 of the node snapshot file. (optional)",
+    help="With sha256 provided integrity check will be performed for you.\n"
+    "Also, download resume option will be available.",
```

Comment:
```suggestion
    prompt="Provide the sha256 of the node snapshot file. (optional)",
    help="With sha256 provided, an integrity check will be performed for you.\n"
    "Also, it will be possible to resume incomplete snapshot downloads.",
```